src/components/home.js

```diff
@@ -109,20 +111,20 @@ class Home extends React.Component {
         };
     };
 
-    handleListChange = (name, value, state, index) => {
-        let newState = state.slice();
-        newState[index] = value;
-        this.setState({
-            [name]: newState
-        });
-    };
+    // handleListChange = (name, value, state, index) => {
```

Comment:
Do we need to keep this around?

Reply:
I don't think it hurts to keep this around. This was the WIP code for changing multiple statuses at once.